contracts/core/Allo.sol

```diff
@@ -114,6 +117,8 @@ contract Allo is
 
         // Set the base fee
         _updateBaseFee(_baseFee);
+
+        _trustedForwarder = __trustedForwarder;
```

Comment:
I think we should be able to update this, wdyt?

Reply:
In the OZ implementation `_trustedForwarder` is immutable, I think they did it for some reason. But I also think we can add a function to update.